exporter/containerimage/exptypes/attestations.go

```diff
@@ -0,0 +1,16 @@
+package exptypes
+
+const (
+	MediaTypeDockerSchema2AttestationType = "application/vnd.in-toto+json"
+	DockerAnnotationReferenceType         = "vnd.docker.reference.type"
+	DockerAnnotationReferenceDigest       = "vnd.docker.reference.digest"
+	DockerAnnotationReferenceDescription  = "vnd.docker.reference.description"
+)
+
+const (
+	// MediaTypeAttestationType       =  "application/vnd.oci.attestation.v1+json"
+	MediaTypeAttestationType          = MediaTypeDockerSchema2AttestationType
+	OCIAnnotationReferenceType        = "org.opencontainers.reference.type"
+	OCIAnnotationReferenceDigest      = "org.opencontainers.reference.digest"
+	OCIAnnotationReferenceDescription = "org.opencontainers.reference.description"
```

Comment:
Not sure what to do about OCI here - or what labels we should attempt to use. Should we consider not supporting attestations for OCI in a first pass, since attaching things like this according to Proposal F isn't standardized?